### PR TITLE
[Shallow] Importing application as module

### PR DESCRIPTION
**Pull request by miguelmaso**
#4629

### CODE REVIEW

**Comment by miguelmaso** [comment]
It passes the tests

**Review by loumalouomega — DISMISSED** [review]

File: applications/ShallowWaterApplication/tests/shallow_water_test_factory.py [hunk 1]
```diff
@@ -3,15 +3,11 @@
 # Importing the Kratos Library
 import KratosMultiphysics
 import KratosMultiphysics.ShallowWaterApplication
-try:
-    import KratosMultiphysics.ExternalSolversApplication
-    have_external_solvers = True
-except ImportError as e:
-    have_external_solvers = False
+import KratosMultiphysics.ExternalSolversApplication
```
Comment (philbucher) [inline applications/ShallowWaterApplication/tests/shallow_water_test_factory.py]: theoretically the linear-sovler-factory should import this

**Review by philbucher — DISMISSED** [review]
perfect, thanks!

File: applications/ShallowWaterApplication/tests/shallow_water_test_factory.py (hunk 1 above)
Comment (miguelmaso) [reply applications/ShallowWaterApplication/tests/shallow_water_test_factory.py]: removed

**Review by philbucher — APPROVED** [review]

**Pull request merged**